src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/PetTsvCreator.java

```diff
@@ -217,7 +217,14 @@ private void setCoveredInterval(String variantChr, int start, int end) {
 
         List<List<String>> rows = new ArrayList<>();
 
-        if (!variant.isReferenceBlock()) {
+        // if the variant is no call, set the GQ to ZERO as "state" in PET table
```

Comment:
this is very minor, but to be totally clear, I think we'd phrase this as "set the pet "state" to GQ ZERO"